apps/src/applab/firebaseStorage.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+/* global Applab */
+
+var FirebaseUtils = require('./firebaseUtils');
+
+/**
+ * Namespace for Firebase storage.
+ */
+var FirebaseStorage = module.exports;
+
+function getKeysRef(channelId) {
+  var kv = FirebaseUtils.getDatabase(channelId).child('storage').child('keys');
+  return kv;
+}
+
+function getRecordsRef(channelId, tableName) {
+  return FirebaseUtils.getDatabase(channelId).child('storage').child('tables')
+    .child(tableName).child('records');
+}
```

Comment:
OMG string interpolation! :-) accessing deeply-nested children is possible this way -- node names cannot contain slashes. this is done now.